drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua

```diff
@@ -14,6 +14,8 @@
 
 local capabilities = require "st.capabilities"
 local supported_values = require "zigbee-multi-button.supported_values"
+local log = require "log"
```

Comment:
Both of these `requires` are unused, and should be removed.

Reply:
done